repository-basex/src/main/java/org/lfenergy/compas/scl/data/basex/BaseXClient.java

```diff
@@ -0,0 +1,459 @@
+// SPDX-FileCopyrightText: 2020 Alliander N.V.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.scl.data.basex;
+
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+
+/**
+ * Java client for BaseX.
+ * Works with BaseX 7.0 and later
+ * <p>
+ * Documentation: https://docs.basex.org/wiki/Clients
+ * <p>
+ * (C) BaseX Team 2005-20, BSD License
+ */
+public final class BaseXClient implements Closeable {
```

Comment:
Maybe creating a story for "How to use this BaseXClient class"?
Do we keep it like it is? Or do we rebuild it?